csunplugged/static/js/website.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+  $("#content-container, #glossary-modal").on("click", ".glossary-term", open_glossary_definition);
+});
+
+// Retrieve glossary definition
+function open_glossary_definition() {
+  // Save glossary slug
+  var slug = $(this).data("glossary-term");
+  // Find glossary modal
+  var glossary_modal = $("#glossary-modal");
+
+  // If glossary modal already has content for term
+  if (glossary_modal.attr("data-glossary-term") == slug) {
+    glossary_modal.modal('show');
+  }
+  // Otherwise download HTML of modal and add to DOM
+  else {
+    // TODO: Allow URL to work for different languages
+    var url = "/topics/glossary/";
+    $.ajax({
+      type: "GET",
+      url: url,
+      data: "term=" + slug,
+      async: true,
+      cache: true,
+      dataType: "json",
+      success: update_glossary_modal
+    });
+  }
+}
+
```

Comment:
lines 1-31 are quite heavily commented, this isn't really necessary unless it's mathematical/extremely specific code, the code should speak for itself, just a docstring is enough (feel free to argue with me)